packages/headless/src/database/mongodb/order/index.ts

```diff
@@ -28,12 +29,18 @@ export class OrderDatabase implements IOrderDatabase {
       return newProductList;
   }
 
-  async getOrderListByUserId(userId: string): Promise<OrderEntity[]> {
-    const orderList = await OrderModel.find({ userId });
-    if (orderList.length > 0) {
-      return orderList;
+  async getOrderListByUserId(userId: string, sortObj: OrderSortQuery): Promise<OrderEntity[]> {
+    const { sortField, sortType} = sortObj;
+    let sortIndex = -1;
+    const sort = {};
+    if(sortField && sortType){
+      if(sortType === 'asc') sortIndex = 1;
+      sort[sortField] = sortIndex;
+    }else{
+      sort['orderedDate'] = sortIndex;
```

Comment:
why hard-coded property?

Reply:
by default orders will be sorted on orderedDate